src/lib/services/feature-toggle-service.ts

```diff
@@ -236,16 +236,23 @@ class FeatureToggleServiceV2 {
             const data = {
                 id: strategy.id,
                 name: strategy.strategyName,
-                featureName: strategy.featureName,
                 constraints: strategy.constraints || [],
                 parameters: strategy.parameters,
             };
+            const preData = {
+                id: existingStrategy.id,
+                name: existingStrategy.strategyName,
+                constraints: existingStrategy.constraints || [],
+                parameters: existingStrategy.parameters,
+            };
```

Comment:
We should have a built-in util to convert the internal strategy configuration to the public representation. Feels cumbersome to manually convert the data.